frontend/src/pages/Requests/RequestItem/RequestItem.tsx

```diff
@@ -23,45 +23,40 @@ import {
 } from './styled';
 import RequestButtons from './RequestButtons';
 
-const RequestItem: React.FC<IRequest> = ({
-  request,
-  approveRequest,
-  declineRequest,
-  approveLoading,
-  declineLoading,
-  isTargetRequest,
-  status,
+interface ReuestItemProps {
+  approveRequest: (requestId: string) => void;
+  approveLoading: boolean;
+  declineRequest: (requestId: string) => void;
+  declineLoading: boolean;
```

Comment:
i think it's order should be fixed like functions are near, boolean too and so on